test/integration/default/verify/controls/aws_vpc.rb

```diff
@@ -0,0 +1,17 @@
+fixtures = {}
+[
+  'ec2_security_group_default_vpc_id',
+].each do |fixture_name|
+  fixtures[fixture_name] = attribute(
+    fixture_name,
+    default: "default.#{fixture_name}",
+    description: 'See ../build/ec2.tf',
+  )
+end
+
+control "aws_vpc default vpc" do
+  describe aws_vpc(fixtures['ec2_security_group_default_vpc_id']) do
```

Comment:
This searches for the default VPC, using the vpc_id that was provided via the fixtures mechanism.  We should also test getting the default VPC by not providing any params.